data_and_models/pipelines/sentence_embedding/Dockerfile

```diff
@@ -37,6 +37,7 @@ RUN git checkout $BBS_REVISION
 # remove ruamel-yaml: https://github.com/pypa/pip/issues/5247#issuecomment-381550610
 RUN rm -rf /opt/conda/lib/python3.8/site-packages/ruamel*
 RUN pip install -r requirements.txt
+RUN pip install -r requirements-data_and_models.txt
```

Comment:
Just not to forget (maybe not necessary to do it), we change the `Dockerfile`, then the corresponding `dvc.lock` are going to change `md5` of this `Dockerfile`.

Reply:
Yes at some point we should definitely re-run the `dvc repro` since the dependencies have indeed changed. Even if the models have not changed, the versions pinned in these `requirements*txt` files have changed, so it is very possible that we do get slightly different results.

On the time _when_ we should do this, I think that this `dvc repro` should be done when we release a new version, which is not the case yet.

Reply:
Sounds a good compromise to do before a release, I did not know about this! Thanks!